src/vs/workbench/contrib/scm/common/scm.ts

```diff
@@ -180,6 +180,9 @@ export interface ISCMViewService {
 
 	readonly menus: ISCMMenus;
 
+	repositories: ISCMRepository[];
+	readonly onDidChangeRepositories: Event<ISCMViewVisibleRepositoryChangeEvent>;
+
 	visibleRepositories: ISCMRepository[];
 	readonly onDidChangeVisibleRepositories: Event<ISCMViewVisibleRepositoryChangeEvent>;
```

Comment:
As discussed earlier, let's create a TODO to address soon and unify these two properties into a single one representing "view model scm repository entries".

Reply:
https://github.com/microsoft/vscode/issues/145568